pkg/controllermanager/controller/shoot/shoot_maintenance_control.go

```diff
@@ -18,6 +18,15 @@ import (
 	"fmt"
 	"time"
 
+	corev1 "k8s.io/api/core/v1"
+	apiequality "k8s.io/apimachinery/pkg/api/equality"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/record"
+	"k8s.io/client-go/util/retry"
+	"github.com/sirupsen/logrus"
```

Comment:
sort imports, CI is failing here

Reply:
ok, that is a job for [importsort](https://github.com/danielfoehrKn/importsort)  :)